querybook/webapp/components/DataDocSchedule/DataDocScheduleForm.tsx

```diff
@@ -284,6 +270,148 @@ export const DataDocScheduleForm: React.FunctionComponent<
     );
 };
 
+function getHelpInfo(selectedWith) {
+    const helpers = {
+        slack: `You can type slack channels here and you can use comma to separate multiple channels`,
+        email: `You can type emails here and you can use comma to separate multiple emails`,
+        teams: `You can type team logins here and you can use comma to separate multiple logins`,
+    };
+
+    return helpers[selectedWith] ?? `You can type here notification recepients`;
+}
+
+const NotifactionFormRow: React.FC<{
+    name: string;
+    index: number;
+    removeRow: (index: number) => void;
+    notificationRow: IDataDocScheduleNotification;
+    notifierOptions: string[];
+    notifyOnOptions: IOptions;
+}> = ({
+    name,
+    index,
+    notificationRow,
+    removeRow,
+    notifierOptions,
+    notifyOnOptions,
+}) => {
+    const { setFieldValue } = useFormikContext<IScheduleFormValues>();
+    const notificationFormName = `${name}[${index}]`;
+    const handleNotifyTo = useCallback((value) => {
+        setFieldValue(
+            `${notificationFormName}.config.to`,
+            value.split(',').map((v) => v.trim())
+        );
+    }, []);
+    const handleRemoveNotifier = useCallback(
+        (index) => removeRow(index),
+        [removeRow]
+    );
+
+    return (
+        <div className="cell-export-field mb24 flex-row">
+            <div className="flex1 mr16">
+                <div className="horizontal-space-between">
+                    <WrappedFormField
+                        label="Notify With"
+                        name={`${notificationFormName}.with`}
+                        type="react-select"
+                        options={notifierOptions}
+                        withDeselect
+                    />
+
+                    <WrappedFormField
+                        label="Notify On"
+                        name={`${notificationFormName}.on`}
+                        type="react-select"
+                        isDisabled={!notificationRow.with}
+                        options={notifyOnOptions}
+                    />
+                </div>
+
+                <SimpleField
+                    label="Notify To"
```

Comment:
In your initial PR, you demoed that you can select a user instead of typing a username. can we make it user can either freely type a username, slack channel name or an email manually, or we can provide a dropdown auto suggestion for built-in users when typing. 

This is a nice to have feature for this PR. we may do this in a separate PR.